monai/transforms/post/array.py

```diff
@@ -183,52 +184,90 @@ class KeepLargestConnectedComponent(Transform):
     """
 
     def __init__(
-        self, applied_values, independent: bool = True, background: int = 0, connectivity: Optional[int] = None
+        self,
+        applied_values=None,
+        independent: bool = True,
+        background: int = 0,
+        connectivity: Optional[int] = None,
+        applied_channel_indices=None,
     ):
         """
         Args:
             applied_values (list or tuple of int): number list for applying the connected component on.
```

Comment:
The type is (list or tuple of int), which is different from `applied_channel_indices`.
Maybe you can refer to other transforms to support (int, list or tuple of int) with `ensure_tuple`?
Then users can use both `applied_values=[1, 2]` and `applied_values=1`.